src/util/native_fs_utils.js

```diff
@@ -263,7 +263,7 @@ async function create_config_file(fs_context, schema_dir, config_path, config_da
         } catch (err) {
             if (err.code !== 'ENOENT') throw err;
         }
-        dbg.log1('native_fs_utils: create_config_file config_path:', config_path, 'config_data:', config_data, 'is_gpfs:', open_mode);
+        dbg.log1('create_config_file:: create_config_file config_path:', config_path, 'config_data:', config_data, 'is_gpfs:', open_mode);
```

Comment:
please don't remove the name of the file from the print

Reply:
dbg print the name of the file, so the result is something like: `...native_fs_utils native_fs_utils:`
changing this will lead to: `...native_fs_utils create_config_file::` which is indicating the function in the file.

Reply:
right, so just remove the double name of function - create_config_file, it appears twice in few different places

Reply:
Ho, a great catch, missed that.